### ISSUE TITLE
ESlint style errors after update

### ISSUE THREAD

**Issue by gihrig** [mxstbr/react-boilerplate#1229]
After dependency update #1225 many new style errors are flagged. I worked around these by disabling the new rules, but, IMO the code should be corrected and the rules re-enabled.

To reproduce, enable the rules listed below in package.json and run `npm run lint`.

Error:
```
~/react-bp/app/app.js
13:1 error Unexpected '!' in 'file?name=[name].[ext]!./favicon.ico'.
Do not use import syntax to configure webpack loaders
```

_eslint rule import/no-webpack-loader-syntax in package.json <--- disabled_
...
```
29:30 error Absolute imports should come before relative imports
```
_eslint rule import/first in package.json <--- disabled_
...
```
47:1 error Import in body of module; reorder to top
```
_eslint rule import/first in package.json <--- disabled_
...
Many other files: "✖ 104 problems (104 errors, 0 warnings)"

**Comment by KarandikarMihir** [comment]
Is this related to #1233 ? @gihrig

**Comment by gihrig** [comment]
@KarandikarMihir No, these errors are legitimate, assuming we accept that webpack dependent imports are undesirable and that the order of imports is important.

> This syntax is non-standard, so it couples the code to Webpack. The recommended way to specify Webpack loader configuration is in a Webpack configuration file. [Ref](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-webpack-loader-syntax.md).

> Notably, imports are hoisted, which means the imported modules will be evaluated before any of the statements interspersed between them. Keeping all imports together at the top of the file may prevent surprises resulting from this part of the spec. [Ref](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/first.md).

**Comment by gihrig** [comment]
Resolved by #1275 

Huge thanks @KarandikarMihir

**Comment by JeaksonChen** [comment]
so?
